### PR TITLE
Remaing Laos EOC visuals

### PR DESCRIPTION
**Pull request by nick-beyondessential**
### Issue #: 2644
- [2644](https://github.com/beyondessential/tupaia-backlog/issues/2644) Map Overlay: Weekly Dengue Cases by District
- [2645](https://github.com/beyondessential/tupaia-backlog/issues/2645) Map Overlay: Weekly Malaria Cases by Province 
- [2647](https://github.com/beyondessential/tupaia-backlog/issues/2647) Dashboard element: Malaria cases bar chart at district and province 

### Changes:
- migrations to add the above visuals

### CODE REVIEW

File: packages/database/src/migrations/20210503024117-AddLaEocMapOverlayDengueCasesByWeekDistrict-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  dataElementCode: 'NCLE_Disease_Name',
+  isDataRegional: false,
+  measureBuilderConfig: {
+    programCodes: ['NCLE_Communicable_Disease'],
+    dataSourceType: 'custom',
+    aggregationType: 'COUNT_PER_ORG_GROUP',
+    dataElementCode: 'NCLE_Disease_Name',
+    aggregationConfig: {
+      condition: {
+        value: ['7.1', '7.2', '7.3'],
```
Comment (rohan-bes) [inline packages/database/src/migrations/20210503024117-AddLaEocMapOverlayDengueCasesByWeekDistrict-modifies-data.js:33]: Can the data element return any value other than one of these three? just curious

File: packages/database/src/migrations/20210503024117-AddLaEocMapOverlayDengueCasesByWeekDistrict-modifies-data.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        operator: 'in',
+      },
+    },
+    entityAggregation: {
+      dataSourceEntityType: 'facility',
+      aggregationEntityType: 'sub_district',
+      aggregationOrder: 'BEFORE',
+    },
```
Comment (rohan-bes) [inline packages/database/src/migrations/20210503024117-AddLaEocMapOverlayDengueCasesByWeekDistrict-modifies-data.js:41]: I believe we can combine the standard aggregation and the entity aggregation into a single aggregation like so:
```suggestion
    entityAggregation: {
      dataSourceEntityType: 'facility',
      aggregationEntityType: 'sub_district',
      aggregationType: 'COUNT_PER_ORG_GROUP',
      aggregationConfig: {
      condition: {
        value: ['7.1', '7.2', '7.3'],
        operator: 'in',
      },
    },
```

File: packages/database/src/migrations/20210503082425-AddLaEocMalariaCasesBarChart-modifies-data.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+const buildReportConfig = (id, entityType) => ({
+  id,
+  dataBuilder: 'countEventsPerPeriod',
+  dataBuilderConfig: {
+    programCode: ['Malaria_Case_Reporting'],
+    dataElement: 'Total_Positive_Malaria_Cases',
+    entityAggregation: {
+      dataSourceEntityType: 'facility',
+      aggregationEntityType: entityType,
```
Comment (rohan-bes) [inline packages/database/src/migrations/20210503082425-AddLaEocMalariaCasesBarChart-modifies-data.js:40]: Am I correct to understand that the only difference between these dashboards is the `aggregationEntityType`? While I wouldn't necessarily push for this right now, we actually have plans to introduce a `requested` `aggregationEntityType` which would change this entity type based on the requested entity! Would be ideal in this scenario.

I've introduced the functionality on the PSSS Phase 2 branch, but that code is a little ways off merging, perhaps lets just add a comment in this migration mentioning we should alter these dashboards once that ticket is merged. Ticket in question: https://github.com/beyondessential/tupaia-backlog/issues/2697

**Review by rohan-bes — CHANGES_REQUESTED** [review]
One small change requested by looks great otherwise!

File: packages/database/src/migrations/20210503082425-AddLaEocMalariaCasesBarChart-modifies-data.js (hunk 3 above)
Comment (nick-beyondessential) [reply packages/database/src/migrations/20210503082425-AddLaEocMalariaCasesBarChart-modifies-data.js:40]: Yes! I've been meaning to ask about that, we have a bunch that differ only by `aggregationEntityType` will be great to empty out some of those nigh duplicate reports!

**Comment by nick-beyondessential** [comment]
Closed for rebased on different branch due to conflicts: https://github.com/beyondessential/tupaia/pull/2504

**Pull request closed without merging**